qute_style/widgets/toggle.py

```diff
@@ -80,19 +78,20 @@ def sizeHint(self) -> QSize:  # pylint: disable=invalid-name
             )
         return QSize(width, QuteStyle.ToggleOptions.BOX_HEIGHT)
 
-    @property
+    @Property(int)
     def position(self) -> int:
         """Return actual position."""
-        return int(self._position)
+        return self._position
 
-    @position.setter
+    # todo: Qt property is not recognized properly atm
+    @position.setter  # type: ignore[no-redef]
     def position(self, pos: int):
         """Set actual position."""
         self._position = pos
         self.update()
 
     @Slot(int, name="setup_animation")
-    def setup_animation(self, value: int) -> None:
+    def setup_animation(self, value: Qt.CheckState) -> None:
```

Comment:
Das hast jetzt den Funktionsparameter auf CheckState aber in der FUnktion wird noch der int verwendet